include/Gaffer/ShufflePlug.h

```diff
@@ -70,6 +70,9 @@ class GAFFER_API ShufflePlug : public ValuePlug
 		BoolPlug *deleteSourcePlug();
 		const BoolPlug *deleteSourcePlug() const;
 
+		BoolPlug *overwritePlug();
```

Comment:
Pretty much everywhere else in Gaffer we're using "replace" as the term for this sort of operation. It might be nice to keep consistency with that, so this might be `replace` or `replaceDestination`. What do you think @murraystevenson, does that make sense?

Reply:
`replaceDestination` would get my vote. I think we only see "overwrite" on MeshType's `overwriteExistingNormals` plug, and as you say, "replace" is the more usual user-facing terminology on MergeScenes, Tweaks nodes, etc.

Reply:
There is not much space, horizontally, in the node editor, "Replace Destination" will be hard to fit in, would just "Replace" be ok?